perfscale_regression_ci/scripts/network/ovn_network_metrics_env.sh

```diff
@@ -0,0 +1,12 @@
+# WORKLOAD_TEMPLATE for custom workload of kube-burner
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+export WORKLOAD_TEMPLATE=${WORKLOAD_TEMPLATE:-"${DIR}/../../kubeburner-object-templates/ovn-network-metrics-config.yaml"}
+# ENVs to overwrite the kube-burner configuration file
+export NAME=${NAME:-"ovn-network-metrics"}
+export NAMESPACE=${NAMESPACE:-"ovn-network-metrics"}
+export JOB_ITERATION=${JOB_ITERATION:-15}
+export QPS=${QPS:-50}
+export BURST=${BURST:-50}
+export POD_REPLICAS=${POD_REPLICAS:=500}
+# Other ENV needed by the test case script
+export NETWORK_POLICY=${NETWORK_POLICY:-"${DIR}/../../content/ovn_metrics_network_policy.yaml"}
```

Comment:
Add an enviornment variable here setting COMPARISON_CONFIG to blank, doesn't look like this test run has that information and by default the [custom workload env variables](https://github.com/openshift/svt/blob/f6fcfd9902b6087f87cdfaaa7152408ecb3b18b2/perfscale_regression_ci/scripts/custom_workload_env.sh#L9) is setting it to clusterversion 

`05-11 14:30:04.255  2023-05-11 18:30:04,196 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'metricName.keyword': 'clusterVersion'}, 'buckets': ['labels.version.keyword'], 'aggregations': {'timestamp': ['max']}}`

Reply:
@paigerube14 I saw in the Jenknis job log, the Metrics profile is set to metrics-profiles/metrics.yaml
`05-12 02:26:51.927  Metrics profile: metrics-profiles/metrics.yaml`
And that metrics profile should be able to collect the clusterVersion metrics 
https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/metrics-profiles/metrics.yaml#L133
I'm curious why the clusterVersion metric was not be captured by the benchmark-comparison from ES